docs/core/configuration.md

```diff
@@ -634,12 +634,12 @@ What happens in reality is, however, a little bit different:
 #4 block - Alice
 ```
 
-That's because Bob is too fast at proposing blocks. This leaves Alice very
+That's because Bob is too fast at committing blocks. This leaves Alice very
```

Comment:
Line 638/639 below is inconsistent. If Bob and Alice have the same voting power, and they are the only validators, both have to vote to commit a block.